api-specs/include/protected_storage.h

```diff
@@ -57,32 +57,39 @@ typedef uint32_t psa_ps_status_t;
 #define PSA_PS_ERROR_INSUFFICIENT_SPACE        4   /**<    The operation failed because there was insufficient space on the storage medium */
 #define PSA_PS_ERROR_STORAGE_FAILURE           6   /**<    The operation failed because the physical storage has failed (Fatal error) */
 #define PSA_PS_ERROR_BAD_POINTER               7   /**<    The operation failed because one of the provided pointers is invalid, for example is `NULL` or references memory the caller cannot access */
-#define PSA_PS_ERROR_KEY_NOT_FOUND             8   /**<    The operation failed because the provided key value was not found in the storage */
+#define PSA_PS_ERROR_UID_NOT_FOUND             8   /**<    The operation failed because the provided key value was not found in the storage */
```

Comment:
The description should be also be changed to UID?

Reply:
Thanks for raising.  I will follow with architect for header file.